llvmlite/binding/orcjit.py

```diff
@@ -272,12 +272,17 @@ def _dispose(self):
         self._capi.LLVMPY_LLJITDispose(self)
 
 
-def create_lljit_compiler(target_machine=None):
+def create_lljit_compiler(target_machine=None,
+                          use_jit_link=False,
+                          suppress_errors=False):
```

Comment:
Probably better to make `use_jit_link` and `suppress_errors` keyword-only arguments to anticipate for future changes. e.g. if we drop `use_jit_link` in the future, `create_lljit_compiler(tm, True)` would mean something else; but the keyword-only will force user to write `create_lljit_compiler(tm, use_jit_link=True)` and the API changes will not be silent.
```suggestion
def create_lljit_compiler(target_machine=None, *,
                          use_jit_link=False,
                          suppress_errors=False):
```

Reply:
Fixed.